doc/DOCUMENTATION.org

```diff
@@ -3389,54 +3389,77 @@ Examples:
-| ~SPC k Dx~  | backward delete expression                                          |
-| ~SPC k e~   | unwrap current expression and kill all symbols after point          |
-| ~SPC k E~   | unwrap current expression and kill all symbols before point         |
-| ~SPC k h~   | previous symbol                                                     |
-| ~SPC k H~   | go to previous sexp                                                 |
-| ~SPC k i~   | switch to =insert state=                                            |
-| ~SPC k I~   | go to beginning of current expression and switch to =insert state=  |
-| ~SPC k j~   | next closing parenthesis                                            |
+| ~SPC k t~   | switch places of expressions left and right of point (transpose)    |
+| ~SPC k ` t~ | hybrid version of transpose sexp (can be used in non lisp dialects) |
 | ~SPC k J~   | join expression                                                     |
-| ~SPC k k~   | previous opening parenthesis                                        |
-| ~SPC k l~   | next symbol                                                         |
-| ~SPC k L~   | go to next sexp                                                     |
-| ~SPC k p~   | paste after                                                         |
-| ~SPC k P~   | paste before                                                        |
-| ~SPC k r~   | raise expression (replace parent expression by current one)         |
-| ~SPC k s~   | forward slurp expression                                            |
-| ~SPC k S~   | backward slurp expression                                           |
-| ~SPC k t~   | transpose expression                                                |
-| ~SPC k u~   | undo                                                                |
-| ~SPC k U~   | got to parent sexp backward                                         |
-| ~SPC k C-r~ | redo                                                                |
-| ~SPC k v~   | switch to =visual state=                                            |
-| ~SPC k V~   | switch to =visual line state=                                       |
-| ~SPC k C-v~ | switch to =visual block state=                                      |
-| ~SPC k w~   | wrap expression with parenthesis                                    |
-| ~SPC k W~   | unwrap expression                                                   |
-| ~SPC k y~   | copy expression                                                     |
+
+****** Adding parts
+
+| Key binding | Description                                          |
+|-------------+------------------------------------------------------|
+| ~SPC k (~   | insert expression before (same level as current one) |
+| ~SPC k )~   | insert expression after (same level as current one)  |
+| ~SPC k w~   | wrap expression with parenthesis                     |
+
+****** Deleting parts
```

Comment:
```org
****** Deletion

| Key binding      | Description                                                                                   |
|------------------+-----------------------------------------------------------------------------------------------|
| Lisp Commands    |                                                                                               |
|------------------+-----------------------------------------------------------------------------------------------|
| ~SPC k ds~       | delete symbol                                                                                 |
| ~SPC k Ds~       | delete symbol backward                                                                        |
| ~SPC k dw~       | delete word                                                                                   |
| ~SPC k Dw~       | delete word backward                                                                          |
| ~SPC k dx~       | delete sexp: ~(as <cursor> a ..) -> (as <cursor> ..)~                                         |
| ~SPC k Dx~       | delete sexp backward: ~(as a <cursor> ..) -> (as <cursor> ..)~                                |
| ~SPC k e~        | unwrap current list and kill all sexps after point: ~(as (bs <cursor> ..) ..) -> (as bs ..)~  |
| ~SPC k E~        | unwrap current list and kill all sexps before point: ~(as (.. <cursor> bs) ..) -> (as bs ..)~ |
| ~SPC k r~        | unwrap current list and kill all sexps except next one: ~(as <cursor> b ..) -> <cursor>(b)~   |
| ~SPC k W~        | unwrap sexp: ~(as) -> as~                                                                     |
|------------------+-----------------------------------------------------------------------------------------------|
| Non-Lisp Comands |                                                                                               |
|------------------+-----------------------------------------------------------------------------------------------|
| ~SPC k ` k~      | delete sexp                                                                                   |
|------------------+-----------------------------------------------------------------------------------------------|
| Note 1:          | ~xs~ is one or multiple sexp, ~x~ is a single sexp                                            |
| Note 2:          | cursor is at ~<cursor>~ when presented, or in the list otherwise                              |
```

Reply:
I changed raise to call it raise and use just the example to explain. similar for slice + kill forward/backward.

Reply:
You need to push your local changes so I can see them...

Reply:
Sorry, usually reviews are not real-time, so I often comment before I push :)